specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-04-01-preview/TenantActivityLogAlerts.json

```diff
@@ -0,0 +1,543 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "description": "Updates 'tags' and 'enabled' fields in an existing Tenant Alert rule. This method is used to update the Alert rule tags, and to enable or disable the Alert rule. To update other fields use CreateOrUpdate operation.",
+        "operationId": "TenantActivityLogAlerts_Update",
+        "x-ms-examples": {
+          "Patch a Tenant Activity Log Alert rule": {
+            "$ref": "./examples/TenantActivityLogAlertRule_UpdateRule.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ManagementGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/TenantActivityLogAlertNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "tenantActivityLogAlertRulePatch",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/TenantAlertRulePatchObject"
+            },
+            "description": "Parameters supplied to the operation."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "An existing Activity Log Alert rule was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/TenantActivityLogAlertResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.AlertsManagement/tenantActivityLogAlerts": {
+      "get": {
+        "description": "Get a list of all Tenant Activity Log Alert rules in the tenant.",
+        "operationId": "TenantActivityLogAlerts_ListByTenant",
+        "x-ms-examples": {
+          "List activity log alerts By Tenant": {
+            "$ref": "./examples/TenantActivityLogAlertRule_ListByTenant.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request succeeded.",
+            "schema": {
+              "$ref": "#/definitions/TenantAlertRuleList"
+            }
+          },
+          "default": {
+            "description": "An error occurred and the list of Activity Log Alert rules could not be retrieved.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
```

Comment:
>ErrorResponse [](http://example.com/codeflow?start=37&length=13)

There's error response defined in common-types too, R U sure you want to redefine it?